libs/ui/select/brain/src/lib/tests/select-reactive-form.ts

```diff
@@ -96,6 +96,46 @@ export class SelectSingleValueWithInitialValueTestComponent {
 	form = new FormGroup({ fruit: new FormControl('apple') });
 }
 
+@Component({
+	standalone: true,
+	imports: [CommonModule, FormsModule, ReactiveFormsModule, BrnSelectImports],
+	// eslint-disable-next-line @angular-eslint/component-selector
+	selector: 'select-reactive-field-fixture',
+	template: `
+		<form [formGroup]="form">
+			<brn-select class="w-56" formControlName="fruit" placeholder="Select a Fruit">
+				<button brnSelectTrigger data-testid="brn-select-trigger">
+					<brn-select-value data-testid="brn-select-value" />
+				</button>
+				<brn-select-content class="w-56" data-testid="brn-select-content">
+					<label brnSelectLabel>Fruits</label>
+					<div brnOption value="apple">Apple</div>
+					<div brnOption value="banana">Banana</div>
+					<div brnOption value="blueberry">Blueberry</div>
+					<div brnOption value="grapes">Grapes</div>
+					<div brnOption value="pineapple">Pineapple</div>
+					<div>Clear</div>
+				</brn-select-content>
+			</brn-select>
+			@if (form.controls.fruit.invalid && form.controls.fruit.touched) {
+				<span class="text-destructive">Required</span>
+			}
+		</form>
+	`,
+})
+export class SelectSingleValueWithInitialValueWithAsyncUpdateTestComponent {
+	form = new FormGroup({ fruit: new FormControl('apple') });
+
+	constructor() {
+		// queueMicrotask(() => {
```

Comment:
Should this comment be removed?

Reply:
I left it in case later a bug comes in and we can test it again based on different timing.